frontend/packages/core/src/AppLayout/drawer.tsx

```diff
@@ -151,6 +151,12 @@ const Group: React.FC<GroupProps> = ({
   if (React.Children.count(children) === 0) {
     return null;
   }
+  // TODO (dschaller): revisit how we handle long groups once we have designs.
+  // n.b. this is a stop-gap solution to prevent long groups from looking unreadable.
+  let formattedHeading = heading;
+  if (heading.length > 11) {
+    formattedHeading = `${heading.substring(0, 10)}...`;
+  }
```

Comment:
Nit: This could probably be a ternary, but this is very readable too.

Reply:
👍  I figured making it explicit would call attention to it for refactoring later :)